charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +125,23 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
+  log-handler.sh: |
+    #!/bin/bash
+    ENDPOINT="{{ $.Values.webdav.portalNotificationEndpoint }}"
+    while IFS= read -r line; do
+      read -a arr <<< $line
+      if [[ ("${arr[1]}" == "PUT" || "${arr[1]}" == "DELETE") && "${arr[2]}" =~ ^/webdav.* ]] then
+        OP="Unknown"
+        if [[ "${arr[0]}" == "201" && "${arr[1]}" == "PUT" ]] then
+          OP="Create"
+        elif [[ "${arr[0]}" == "204" && "${arr[1]}" == "PUT" ]] then
+          OP="Update"
+        elif [[ "${arr[0]}" == "204" && "${arr[1]}" == "DELETE" ]] then
+          OP="Delete"
+        fi
+        echo "Send notification to $ENDPOINT: $OP ${arr[2]}" > /proc/self/fd/1
+        #wget -S ${ENDPOINT} 2>&1 | grep "HTTP/" | awk '{print $2}'
```

Comment:
I think wget is primarily a tool to download stuff and less commonly used. I'd prefer curl:

Sth. like this:
```bash
RES_CODE=$(curl -sw "%{http_code}\n" -o /dev/null ${ENDPOINT})
```
Not sure if alpine (`ash`) is sufficient, but `quay.io/curl/curl` could be a good image.

Reply:
Done